### PR TITLE
Problem Suggestion: Add "Connecting Universities" to Silver - Additional Practice for USACO Silver

### PR DESCRIPTION
**Pull request by maggieliu05**
User `FITX0BI9Ucev9dNM2sQknMZmYsH3` suggested adding the problem [Connecting Universities](https://codeforces.com/contest/700/problem/B) to the `problems` table of the module [Silver - Additional Practice for USACO Silver](https://usaco.guide/silver/silver-conclusion#problemlist-problems).

**Automatically Generated JSON:**
```json
{
  "uniqueId": "cf-700B",
  "name": "Connecting Universities",
  "url": "https://codeforces.com/contest/700/problem/B",
  "source": "CF",
  "difficulty": "Normal",
  "isStarred": false,
  "tags": [
    "DFS"
  ],
  "solutionMetadata": {
    "kind": "autogen-label-from-site",
    "site": "CF"
  }
}
```
**Additional Notes**:
The implementation of the problem is quite simple, although the idea to solve it can be trick

*This PR was automatically generated from a user-submitted problem suggestion on the USACO guide.*

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
@envyaims your call

File: content/3_Silver/Conclusion.problems.json [hunk 1]
```diff
@@ -241,6 +241,19 @@
         "site": "CF"
       }
     },
+    {
+      "uniqueId": "cf-700B",
+      "name": "Connecting Universities",
+      "url": "https://codeforces.com/contest/700/problem/B",
+      "source": "CF",
+      "difficulty": "Normal",
```
Comment (envyaims) [inline content/3_Silver/Conclusion.problems.json:249]:
```suggestion
      "difficulty": "Hard",
```

**Review by envyaims — DISMISSED** [review]
good problem

File: content/3_Silver/Conclusion.problems.json (hunk 1 above)
Comment (envyaims) [reply content/3_Silver/Conclusion.problems.json:249]: oh yea change the ordering too 💀

Comment (SansPapyrus683) [reply content/3_Silver/Conclusion.problems.json:249]: wow very rare of you to raise the difficulty of a problem

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**